guild.go

```diff
@@ -258,44 +196,40 @@ func (c *Client) ModifyGuildChannelPositions(id objects.Snowflake, params []*Mod
 		return err
 	}
 
-	res, err := c.request(&request{
-		method:      http.MethodPatch,
-		path:        fmt.Sprintf(GuildChannelsFmt, id),
-		contentType: JsonContentType,
-		body:        data,
-		reason:      reason,
-	})
-	if err != nil {
-		return err
-	}
+	return NewRequest().
+		Method(http.MethodPatch).
+		Path(fmt.Sprintf(GuildChannelsFmt, id)).
+		ContentType(JsonContentType).
+		Body(data).
+		Reason(reason).
+		Expect(http.StatusNoContent).
+		Send(c)
+}
 
-	if err = res.ExpectsStatus(http.StatusNoContent); err != nil {
-		return err
-	}
+func (c *Client) ListActiveThreads(id objects.Snowflake) ([]*ListThreadsResponse, error) {
+	channels := []*ListThreadsResponse{}
+	err := NewRequest().
+		Method(http.MethodGet).
+		Path(fmt.Sprintf(GuildThreadsFmt, id)).
+		ContentType(JsonContentType).
+		Expect(http.StatusOK).
+		Bind(channels).
```

Comment:
We need to pass a pointer to the slice here, otherwise it will automatically return an error.